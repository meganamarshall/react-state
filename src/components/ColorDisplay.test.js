import React from 'react';
import { shallow } from 'enzyme';
import ColorDisplay from './ColorDisplay';

describe('ColorDisplay component', () => {
  it('renders the color display', () => {
    const wrapper = shallow(<ColorDisplay color='red'/>);
    expect(wrapper).toMatchSnapshot();
  });
});
