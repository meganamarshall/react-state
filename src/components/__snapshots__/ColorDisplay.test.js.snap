// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ColorDisplay component renders the color display 1`] = `
<div
  className="ColorDisplay"
  style={
    Object {
      "backgroundColor": "red",
    }
  }
/>
`;
